pkg/helper/roles/helpers.go

```diff
@@ -23,6 +23,19 @@ const (
 	RosaUpgradeAccRolesModeAuto = "ROSAUpgradeAccountRolesModeAuto"
 )
 
+// GeOperatorRolePrefixFromClusterName returns a valid operator role prefix from the cluster name
+// An operator role prefix is considered valid if it's length is less than or equal to 32 chars.
+// A random 4 characters label is attached to the cluster name to reduce chances of collision.
+// The cluster name and the random label are separate by '-'.
+// If the cluster name is longer than 27 characters, only the first 27 characters will be used.
+func GeOperatorRolePrefixFromClusterName(clusterName string) string {
+	maxClusterLengthToUse := 27
```

Comment:
@machi1990 Nit: Shall we make this a `const`?

Reply:
done